README.md

```diff
@@ -1,92 +1,40 @@
-# waterthetrees
+# Water the Trees - Frontend
 
-Water The Trees is a platform that crowd sources tree planting and maintenance. We believe in the power of trees to restore natural habitat for animals, insects, and fauna. We are interesting in continuous massive tree planting events to help sequester carbon and stabilize climate change extremes.
+Water the Trees (WTT) is a platform for crowd-sourced tree-planting and maintenance. We believe in the power of trees to restore natural habitats for wildlife and sequester carbon to stabilize the effects of climate change. 
```

Comment:
A description of the project should probably reside in the main repo's readme, so it doesn't have to be repeated in each child repo.  (Though it's currently copy/pasted a bunch of places.)